fairscale/optim/adascale.py

```diff
@@ -514,8 +514,9 @@ def add_param_group(self, pg: Dict) -> None:
         # Extend the states.
         for name in self._state.keys():
             assert name.startswith("grad_sqr_avg") or name.startswith("grad_var_avg"), name
-            if isinstance(self._state[name], int):
-                # This is the "_count" variable.
+            if name.endswith("_count"):
```

Comment:
hmm, is that expected ? looks like a logic change

Reply:
right, it is induced by mypy checking. newer numpy version actually enabled mypy to find out that adascale code here had a mixed typing, so it flagged it. I fixed it by change the typing to not be mixed. So, it is a typing change. No logic changes.